AzFilesHybrid/AzFilesHybrid.psm1

```diff
@@ -3359,28 +3359,18 @@ function Test-Port445Connectivity
     [CmdletBinding()]
 
     param (
-        [Parameter(Mandatory=$True, Position=0, HelpMessage="Storage account name")]
-        [string]$StorageAccountName,
-
-        [Parameter(Mandatory=$True, Position=1, HelpMessage="Resource group name")]
-        [string]$ResourceGroupName
+        [Parameter(Mandatory=$True, Position=0, HelpMessage="Account FileEndPoint")]

```

Comment:
While we're at it here, I would suggest improving the help message. The help message is what is printed if you do `Get-Help Test-Port445Connectivity`. Let's not be scared to be verbose here. What about this?

> The DNS name of the storage account's File endpoint, e.g., "myAccount.file.core.windows.net".